samples/browser/src/main/java/org/mozilla/samples/browser/BrowserFragment.kt

```diff
@@ -145,11 +146,14 @@ class BrowserFragment : Fragment(), BackHandler {
             owner = this,
             view = layout)
 
+        val readerViewFeature = ReaderViewFeature(components.engine, layout.toolbar, components.sessionManager)
```

Comment:
nit: Since this keeps a strong reference to the View which may have a shorter lifetime than the fragment, we may want to wrap this in `ViewBoundFeatureWrapper`.

Reply:
Ah good point. Fixed!